index.js

```diff
@@ -11,6 +12,7 @@ function wabbitzzz (opt) {
 		request: request(opt),
 		response: response(opt),
 		rpc: rpc(opt),
+		Connection: connection(opt),
```

Comment:
I think you need to add the Connection object as a property of wabbitzzz as well (like after line 23)